src/libraries/System.Runtime.InteropServices/gen/Microsoft.Interop.SourceGeneration/Resources/xlf/Strings.de.xlf

```diff
@@ -103,8 +103,8 @@
         <note />
       </trans-unit>
       <trans-unit id="MarshallingStringOrCharAsUndefinedNotSupported">
-        <source>Marshalling string or char without explicit marshalling information is not supported. Specify either 'LibraryImportAttribute.CharSet' or 'MarshalAsAttribute'.</source>
-        <target state="translated">Marshalling von Zeichenfolgen oder Zeichen ohne explizite Marshallinginformationen wird nicht unterstützt. Geben Sie entweder \"LibraryImportAttribute.CharSet\" oder \"MarshalAsAttribute\" an.</target>
+        <source>Marshalling string or char without explicit marshalling information is not supported. Specify 'LibraryImportAttribute.StringMarshalling', 'LibraryImportAttribute.StringMarshallingCustomType', 'MarshalUsingAttribute' or 'MarshalAsAttribute'.</source>
+        <target state="new">Marshalling von Zeichenfolgen oder Zeichen ohne explizite Marshallinginformationen wird nicht unterstützt. Geben Sie entweder \"LibraryImportAttribute.StringMarshalling\" oder \"MarshalAsAttribute\" an.</target>
```

Comment:
Am I following the correct protocol for these files by flipping the updated string to `state="new"`?

Reply:
I _think_ so, but honestly all one should need to do is rebuild the product locally and the build should update these appropriately. That was the guidance from @elinor-fung I thought.

Reply:
Boo. It would appear the correct state is `needs-review-translation`.

See https://github.com/dotnet/runtime/pull/67863 from when @elinor-fung was fixing my mistake :(

Reply:
Thanks, I did not know that the build will update these for me.

Reply:
Yep, as Aaron said, a (non-CI) build will automatically update the .xlf files by default. You can also explicitly just execute the target to update the files: `dotnet build -t:UpdateXlf`.